pos_subscriber_topic: /k4a/body_tracking_data
scan_subscriber_topic: /k4a/scan
feedback_subscriber_topic: /arduino/feedback